joomla/cassiopeia
# Sample Data - Banner and Newsflash

Pull Request for Issue #3 .

### Summary of Changes
The purpose of these changes in sample data is giving testers and developers some data for testing chenges on cassiopeia.Different layouts, especially the module classes can be prsented

The first step in sample data were some addional menuItems to present a  drop down  menu ( metismenu)
This second steps adds makes more changes.

- Add some text and a button in the banner overlay the banner area.
- Add some intro images to articles
- Make the MenuItem "Blog" a Link to a Blog in mansonry layout
- Some modules   which are not < nav > are tagged as < aside >  (a11y) see  chart in comment https://github.com/joomla/cassiopeia/pull/55#issuecomment-685997907
- A **newflash module** has been added. There are three horizontal boxes in position top-a.
- These articles have an intro-image, at the moment it is the banner image.
- Blog featurd (Template) is in a two-columns layout with one leading article, using modifier classes for blog-layout..
- Newslash and banner are only on the homepage.


<img width="713" alt="cassiopeia2" src="https://user-images.githubusercontent.com/1035262/92323852-42f0c200-f03c-11ea-9800-1652c761f0c3.PNG">


### Testing Instructions
Install blog sample data on a fresh installation. 
Make sure that you have the newest version and made npm ci for the horizontal layout of the newsflash boxes.


### Note
Sample blog data will be changed permanently to show important and attractive features, so will be updated during the whole development of cassiopeia.
